drivers/pwm/pwm_mchp_xec_bbled.c

```diff
@@ -344,6 +408,12 @@ static int pwm_bbled_xec_init(const struct device *dev)
 	return 0;
 }
 
+#ifdef CONFIG_PM_DEVICE
+#define XEC_BBLED_PM_DEV_DEFINE(i) PM_DEVICE_DT_INST_DEFINE(i, pwm_bbled_xec_pm_action);
+#else
+#define XEC_BBLED_PM_DEV_DEFINE(i) /* Not used */
+#endif
```

Comment:
`PM_DEVICE_DT_INST_DEFINE()` does not need to be wrapped in `#fdef CONFIG_PM_DEVICE`.

Reply:
Ok, updated.